### PR TITLE
nfs service name did not match to env variable in statefulset

### PR DESCRIPTION
**Pull request by vtuson**
**What this PR does / why we need it**:
Chart wont work as it is, since the pod is trying to contact as svc that doesnt exist

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
#5271 

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/nfs-server-provisioner/Chart.yaml [hunk 1]
```diff
@@ -2,12 +2,13 @@ apiVersion: v1
...
 name: nfs-server-provisioner
-version: 0.1.0
+version: 0.1.1
 maintainers:
 - name: kiall
   email: kiall@macinnes.ie
+home: https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
 sources:
-- https://github.com/kubernetes-incubator/external-storage/tree/master/nfs
+- https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
```
Comment (prydonius) [inline stable/nfs-server-provisioner/Chart.yaml]: can you keep the previous source, this is usually used for linking to Dockerfiles and other sources and not really this repo

File: stable/nfs-server-provisioner/templates/statefulset.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ spec:
                 fieldRef:
                   fieldPath: status.podIP
             - name: SERVICE_NAME
-              value: {{ .Chart.Name }}
+              value: {{ template "nfs-provisioner.fullname" . }}
```
Comment (prydonius) [inline stable/nfs-server-provisioner/templates/statefulset.yaml:59]: great catch!

**Review by prydonius — APPROVED** [review]

File: stable/nfs-server-provisioner/Chart.yaml (hunk 1 above)
Comment (vtuson) [reply stable/nfs-server-provisioner/Chart.yaml]: Ah, yes. I miss read the url

Comment (vtuson) [reply stable/nfs-server-provisioner/Chart.yaml]: I will fix tomorrow

Comment (vtuson) [reply stable/nfs-server-provisioner/Chart.yaml]: fixed :)

File: stable/nfs-server-provisioner/templates/statefulset.yaml (hunk 2 above)
Comment (kiall) [reply stable/nfs-server-provisioner/templates/statefulset.yaml:59]: Thanks! I must have missed this!

**Pull request merged**